### PR TITLE
Time Series

### PR DESCRIPTION
**Pull request by matthewma7**
This PR is based on @manthey work on the time series. tests and documentation added.

The sample Time Series David created for testing (rename extension to geojson before uploading)
[sample-timeseries.geojson.txt](https://github.com/Kitware/minerva/files/964990/sample-timeseries.geojson.txt)

### CODE REVIEW

**Comment by aashish24** [comment]
this is great @matthewma7 

@dorukozturk could you review this branch?

**Comment by dorukozturk** [comment]
@aashish24 Sure, I will review it today.

**Comment by ronichoudhury** [comment]
@matthewma7 do you mind posting a screencap of this work?

**Comment by matthewma7** [comment]
@ronichoudhury Sure, screencap uploaded.

**Comment by ronichoudhury** [comment]
Thanks! Looks cool!

File: plugin_tests/client/mockVGL.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
+/* eslint-disable camelcase */
+/* eslint-disable underscore/prefer-constant */
```
Comment (aashish24) [inline plugin_tests/client/mockVGL.js]: @matthewma7 with recent release of geojs, we no longer need to maintain a copy of this 

@manthey

Comment (aashish24) [inline plugin_tests/client/mockVGL.js]: but we could do it as another PR since you will need to bump up the version of geojs

Comment (matthewma7) [inline plugin_tests/client/mockVGL.js]: @ronichoudhury I see, I have another PR to bump the version. I explicitly added this file because I encountered errors when writing the tests.

Comment (aashish24) [inline plugin_tests/client/mockVGL.js]: roger that @matthewma7 and it was me not @ronichoudhury 😄

Comment (matthewma7) [inline plugin_tests/client/mockVGL.js]: @aashish24 Oh, sorry, keyboard mistake. I know you left the message.
